_specs/ecip-_.md

```diff
@@ -0,0 +1,103 @@
+---
+ecip: TBD
+title: The Meta Fork ECIP: Establishing _Fork_ type ECIPs and their specifications and processes
+author: Mr. Meows D. Bits
+discussions-to: TBD
+status: WIP
+type: Meta
+created: 2019-11-29T14:17:42-05:00
+resolution: TBD
+---
+
+### Abstract
+
+Establishing _Fork_ type ECIPs, and their standards and processes.
+
+#### TL;DR
+
+Make ECIPs with `Type` _Fork_ a thing. _Fork_ ECIPs should follow this process:
+- A "shell" Fork ECIP is opened, fitting the [template provided](TODO). This ECIP __does not initially include `Activation Block Number` or `Features` specifications__.
+- While in `WIP` or `Draft` status, one or more change sets are proposed against this document (eg. via Github Pull Requests) __modifying BOTH `Activation Block Number` and `Features` specifications__.
+- ... Stuff happens; there are comments, emails, meetings, bribes, blogs, bragging, trolling, haranguing, arguing, debating, pondering, editing, compromising, constructive criticiziing, etc...
+- One change set is merged to modify the Fork ECIP, yielding a COMPLETE and UNIQUE hard fork specification document.
+
+### Definition
+
+Fork ECIPs are defined as ECIPs specifying PROTOCOL ACTIVATION of any one or more Standards-Core track ECIP(s) at a specified activation block number.
+
+#### Standards
+
+Fork ECIPs should contain only placeholder information for ECIP-sets and block activation numbers while in `WIP` or `Draft` status. 
+
+Modification to any of `Last Call`, `Accepted`, `Final`, or `Active` statuses should be accompanied by the introduction (merge) of a single change set containing COMPLETE and UNIQUE definitions of the placeholder values.
```

Comment:
I think Fork ECIPs should not have "Active" status. They should be "Final", as once it's accepted, no substance change should ever happen to the specification.

Reply:
Resolved with https://github.com/ethereumclassic/ECIPs/pull/224/commits/b00bdcacc07bae5436a1444f2c781c8f49508fce